steps/id-and-residency/carer.tsx

```diff
@@ -259,9 +246,10 @@ const step: ProcessStepDefinition<ResidentDatabaseSchema, "carer"> = {
       ComponentWrapper.wrapDynamic(
         new DynamicComponent({
           key: "carer-live-in-start-date",
-          Component: DateInput,
+          Component: TextInput,
           props: {
             name: "carer-live-in-start-date",
+            label: "Record at least month and year if possible eg MM / YYYY.",
```

Comment:
This is a hint, not a label. the label is the thing you've made a heading above.

Reply:
Ok...just the new design a heading and this as a sentence above the box. Hint doesn't exist on Text Input. what else could i use?

Reply:
Ah. Ok, that's annoying. I forget if `TextInput` supports passing an id for a label, in which case, use the heading, add a paragraph and then this. The label is the heading, not this help text.

Reply:
Ok, it doesn't. I think the work around for right now is to move the heading into this label and put the help text underneath as a paragraph. Not the best UX, but fixing this isn't a quick fix (and is a side effect of not having built these inputs in `lbh-frontend-react` properly).